packages/client/src/arpa_reporter/views/Home.vue

```diff
@@ -98,14 +89,14 @@ export default {
     clearAlert() {
       this.alert = null;
     },
-    downloadAuditReportURL() {
-      return apiURL('/api/audit_report');
-    },
     async sendAuditReport() {
       this.sending = true;
 
       try {
-        const result = await getJson('/api/audit_report?queue=true');
+        const periodId = this.$store.getters.viewPeriod.id || 0;
+        const periodIdQuery = (periodId && !this.viewingOpenPeriod) ? `&period_id=${periodId}` : '';
+        const url = `/api/audit_report?queue=true${periodIdQuery}`;
+        const result = await getJson(url);
```

Comment:
Assembling URLs by hand makes me nervous 😅 

```suggestion
        const query = new URLSearchParams({ queue: true });
        if (periodId && !this.viewingOpenPeriod) { query.set('period_id', periodId); }
        const result = await getJson(`/api/audit_report?${query}`)
```